index.html

```diff
@@ -17,6 +17,9 @@
   <title>SHOP</title>
 
   <link rel="shortcut icon" sizes="32x32" href="/images/shop-icon-32.png">
+  <meta name="google-signin-client_id" content="569649340112-da0ku71mend03mdsu9h7skmv33r2qkg2.apps.googleusercontent.com">
+  <script src="https://apis.google.com/js/platform.js"></script>
```

Comment:
@agektmr, could you please explain why you don't use https://beta.webcomponents.org/element/GoogleWebComponents/google-signin/google-signin element? It uses `google-js-api` under the hood.